### PR TITLE
s/Creator Node/Content Node

### PR DESCRIPTION
**Pull request by raymondjacobson**
Replaces verbiage.
Note-- SPFactoryClient still understands content node as "creator-node" so we pass that string through. That was the only place that broke with find/replace I believe.

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export type Amount = BigNumber
 
 export enum ServiceType {
   DiscoveryProvider = 'discovery-provider',
-  CreatorNode = 'creator-node'
+  ContentNode = 'creator-node'
```
Comment (raymondjacobson) [inline src/types.ts:17]: this guy

Comment (dmanjunath) [reply src/types.ts:17]: i believe this is the only instance of something hardcoded in the contracts

**Review by dmanjunath — APPROVED** [review]
Looks good!

File: src/types.ts (hunk 1 above)
Comment (jowlee) [reply src/types.ts:17]: Yeah, I don't think it's actually even hardcoded in the contracts. 
We just need to make a tx with the serviceTypeManager contract to register the content-node and de-register the creator-node, then re-register the services.
But, I asked roneil and he said it was fine just changing it in the UI for now.

**Review by jowlee — APPROVED** [review]

**Pull request merged**